### DISCUSSION TITLE
Is Release Candidate (rc) stable?

### DISCUSSION THREAD

**Discussion by KrisLau** [react-native-elements/react-native-elements#3561] [Q&A] [answered]
Thanks for all the great work on the library! I recently noticed all the 4.0.0 releases and I wasn't sure if it is recommended to upgrade to v4 now or wait until the official release of v4? Also when is the v4 expected to be released? 

I read the migration guide here: https://reactnativeelements.com/blog/2022/05/15/rneui-migration-guide but I was wondering if the previously downloaded package `react-native-elements` would need to be removed completely and if that is going to be the way even after v4 is released

**Comment by KrisLau** [comment]
UPDATE: Just upgraded it to get rid of the issue I was having with Slider not re-rending when props change and it seems to work pretty well so far

**Comment by arpitBhalla** [comment, marked as answer]
You can remove `react-native-elements`, once `@rneui/*` is released as stable (v4.0.0, not rc). Also `rc` releases are somewhat stable, you can help us by installing it and providing feedback/issues.

**Reply by KrisLau** [reply]
I've been using rc-4 for about a week now and its been great, thanks for all the amazing work!
